### DISCUSSION TITLE
Resizing storage volumes

### DISCUSSION THREAD

**Discussion by mflynn-lanl** [microbiomedata/nmdc-edge#293] [General]
@edwins Is it possible to increase the size of storage volumes? We have seen a big increase in storage usage for the current nmdc-edge. SDSC has been able to increase our storage on the fly. For Jetstream2 in the past, if I ran out of storage I needed to create a new larger volume and copy the data to the new volume. We're currently using 27TB with 10TB being added in the last month. It would be great if I didn't have to create a huge volume (100TB) so that we have enough room for future growth.

**Comment by edwins** [comment]
Let me research into magic castle, but my gut says that there's likely not an easy way to allow for this.
